### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by DelhinRharl**
## In this pull request,

- [ ] I Added tests  for the requests

### CODE REVIEW

File: spec/requests/posts_request_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    it 'the response includes right text' do
+      expect(response.body).to include('The list of post for given specific users\' post id')
+    end
+
+    it 'the correct response status' do
+      expect(response.status).to eq(200)
+    end
+  end
+end
```
Comment (ahmetbozaci) [inline spec/requests/posts_request_spec.rb]: - [ ] I am getting two errors . Please fix the error.

image

File: spec/requests/users_request_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    it 'the response includes right text' do
+      expect(response.body).to include('The list of users\'s id')
+    end
+
+    it 'the correct status' do
+      expect(response.status).to eq(200)
+    end
+  end
+end
```
Comment (ahmetbozaci) [inline spec/requests/users_request_spec.rb]: - [ ]  I am getting two errors. Please fix the error.


image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @DelhinRharl  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- You have one commit for everything. For best practices, you should commit often. [Git Commit Best Practices](https://www.perforce.com/blog/vcs/git-best-practices-git-commit)

___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by andersonlebon — APPROVED** [review]
## Hi @DelhinRharl 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**